__tests__/autocomplete.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { TouchableOpacity } from "react-native";
+import AutoComplete from "../src/components/search/autocomplete.component";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+
+let mockPOIs = [
```

Comment:
BLOCKING:
No need to mock constant data. You can compare them directly. Please change to use the POI data file, so it's easier to see if tests failed due to different POI structure.
See example here: https://github.com/RGPosadas/WayFinder/blob/TASK-10/setup-detox/__tests__/custom-marker.test.js#L54

Reply:
Other solution would be to convert tests to typescript. might just be this file for now or could be done in a seperate issue.

Reply:
Converted this test to typescript